deploy/buildtools/buildconfig.py

```diff
@@ -26,6 +26,8 @@ class BuildConfig:
         deploytriplet: Deploy triplet override.
         launch_time: Launch time of the manager.
         PLATFORM_CONFIG: Platform config to build.
+        frequency: Frequency for the FPGA build. 
```

Comment:
Lets pick a more specific name for this since people tend to get confused. `fpga_freqency` or `host_fpga_frequency` perhaps?

Reply:
This is marked as resolved but it's not changed in the diff. Have you pushed?